src/test/java/fi/metatavu/soteapi/wnspusher/WnsPusherNotificationControllerTest.java

```diff
@@ -0,0 +1,165 @@
+package fi.metatavu.soteapi.wnspusher;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.util.function.Supplier;
+
+import org.apache.http.ParseException;
+import org.apache.http.StatusLine;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.util.EntityUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+import org.slf4j.Logger;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import fi.metatavu.soteapi.settings.SystemSettingController;
+
+public class WnsPusherNotificationControllerTest {
+
+  private static final String TEST_MESSAGE = "TEST_MESSAGE";
+  private static final String TEST_TITLE = "TEST_TITLE";
+  private static final String TEST_APP = "TEST_APP";
+  private static final String TEST_API_KEY = "TEST_API_KEY";
+  private static final String TEST_ENDPOINT = "TEST_ENDPOINT";
+
+  private SystemSettingController systemSettingController;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "systemSettingController" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)